src/services/ui/src/features/submission/app-k/index.tsx

```diff
@@ -70,7 +87,7 @@ export const AppKSubmissionForm = () => {
 
   return (
     <C.SimplePageContainer>
-      {submission.isLoading && <C.LoadingSpinner />}
+      {(submission.isLoading || isDataPolling) && <C.LoadingSpinner />}
```

Comment:
re: hookifying -- looks like a callback func and this state var are your two hook returns

```
const { poll, isPolling } = useSubmissionPoller()
```

Reply:
your `poll` func will be what takes params i'm assuming, since that's where you need to plug them into the poller obj

Reply:
disregard as the extra state requirement is outgoing with form refactors!